src/test/groovy/uk/gov/justice/digital/hmpps/iis/pages/EditDobPage.groovy

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.iis.pages
+
+import geb.Page
+import uk.gov.justice.digital.hmpps.iis.modules.ErrorsModule
+import uk.gov.justice.digital.hmpps.iis.modules.HeaderModule
+import uk.gov.justice.digital.hmpps.iis.modules.SearchFormModule
+
+class EditDobPage extends Page {
```

Comment:
Rather than add a variation of the page where only the url changes, can pass in the url (or a type enum) in constructor. see http://www.gebish.org/manual/0.12.0/#parametrized-pages

eg
```at(new DobPage(mode: 'edit'))```
or something